packages/clients/textLocator/scripts/injectFlag.sh

```diff
@@ -0,0 +1 @@
+sed -i 's/<link/<link data-polar="true"/g' ./dist/index.html
```

Comment:
DISH and this client both still need it for the creation of a website package since their respective .css files contain cache-busting suffixes. This issue has not yet been adressed and it seems out of scope to address it here.